src/main/java/ru/clevertec/userservice/user/controller/UserController.java

```diff
@@ -2,79 +2,73 @@
 import ru.clevertec.userservice.user.model.request.UpdateUserRequest;
-import ru.clevertec.userservice.user.model.response.ApiResponse;
 import ru.clevertec.userservice.user.service.UserService;
 
 import java.util.List;
 import java.util.UUID;
 
 @RestController
 @Slf4j
-@RequestMapping("/api/v1/users")
+@RequestMapping("/api/v1/admin/users")
 @RequiredArgsConstructor
 public class UserController {
     private final UserService userService;
 
     @PostMapping
-    public ApiResponse<UserDto> createUser(@RequestBody CreateUserRequest request) {
-
+    public ResponseEntity<UserDto> createUser(@RequestBody CreateUserRequest request) {
         log.info("###POST createUserRequest = {}", request);
-
         UserDto userDto = userService.saveUser(request);
         log.info("userDto = {}", userDto);
-        return new ApiResponse<>(userDto);
+        return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
     @GetMapping("/email/{email}")
-    public ApiResponse<UserDto> getUserByEmail(@PathVariable String email) {
+    public ResponseEntity<UserDto> getUserByEmail(@PathVariable String email) {
         log.info("###GET email = {}", email);
         UserDto userDto = userService.getUserByEmail(email);
         log.info("-------userDto = {}", userDto);
-        return new ApiResponse<>(userDto);
+        return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
     @GetMapping
-    public ApiResponse<List<UserDto>> getAllUsers() {
-
+    public ResponseEntity<List<UserDto>> getAllUsers() {
         log.info("###GET getAllUsers");
-
-        return new ApiResponse<>(userService.getAllUsers());
+        userService.getAllUsers();
+        return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
     @GetMapping("/{id}")
-    public ApiResponse<UserDto> getUserById(@PathVariable UUID id) {
-
+    public ResponseEntity<UserDto> getUserById(@PathVariable UUID id) {
         log.info("###GET fetched id = {}", id);
-
-        return new ApiResponse<>(userService.getUserById(id));
+        userService.getUserById(id);
+        return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
     @PutMapping
-    public ApiResponse<UserDto> updateUser(@RequestBody UpdateUserRequest request) {
-
+    public ResponseEntity<UserDto> updateUser(@RequestBody UpdateUserRequest request, @RequestParam UUID roleId) {
         log.info("###PUT updateUserRequest = {}", request);
-
-        return new ApiResponse<>(userService.updateUser(request));
-
+        userService.updateUser(request, roleId);
+        return new ResponseEntity<>(HttpStatus.OK);
     }
 
     @DeleteMapping("/{id}")
-    public ApiResponse<UserDto> deleteUser(@PathVariable UUID id) {
-
+    public ResponseEntity<UserDto> deleteUser(@PathVariable UUID id) {
         log.info("###DELETE fetched id = {}", id);
-
-        return new ApiResponse<>(userService.deleteUser(id));
+        userService.deleteUser(id);
+        return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
it is better to use 204 status for delete

Reply:
fixed